horace_core/sqw/PixelData/@PixelDataFileBacked/private/set_as_tmp_obj_.m

```diff
@@ -0,0 +1,14 @@
+function obj =set_as_tmp_obj_(obj,filename)
+%SET_AS_TMP_OBJ_ mark file-base of the filebacked object marked for
+% deleteon when object build over the file goes out of scope.
+
+if nargin == 1
+    filename = obj.full_filename;
+end
+if isempty(obj.tmp_file_holder_)
+    obj.tmp_file_holder_ = TmpFileHandler(filename,true);
+else
+    if ~strcmp(obj.tmp_file_holder_.file_name,filename)
+        obj.tmp_file_holder_ = TmpFileHandler(filename,true);
+    end
+end
```

Comment:
Noting this doesn't have the same structure as one above.
```suggestion
else if ~strcmp(obj.tmp_file_holder_.file_name,filename)
    obj.tmp_file_holder_ = TmpFileHandler(filename,true);
end
```

Reply:
? It is a shugar so probably a bit simple than in sqw object. Anyway, for pixels it is used for tests only.
But probably still should be the same as in sqw object